util/settings/settings.go

```diff
@@ -96,6 +98,43 @@ type HelmRepoCredentials struct {
 	KeySecret      *apiv1.SecretKeySelector `json:"keySecret,omitempty"`
 }
 
+type ExcludedResource struct {
+	ApiGroups []string `json:"apiGroups"`
+	Kinds     []string `json:"kinds"`
+	Clusters  []string `json:"clusters"`
+}
+
+func (r ExcludedResource) matchGroup(apiGroup string) bool {
+	for _, excludedApiGroup := range r.ApiGroups {
+		if glob.MustCompile(excludedApiGroup).Match(apiGroup) {
```

Comment:
I guess `glob.MustCompile` panics if provided string is not a valid pattern. The controller should not crash if user provided settings are not valid. I think we should try to parse filters in `updateSettingsFromConfigMap` method and logs error message if provided filters are not valid.

Reply:
This does panic.

Reply:
Hmmmmmm, I'm logging them when invoked, but this isn't great.

Reply:
Options

1. Quietly ignore them (simple and nasty).
2. Log on invocation (simple).
3. Log on load (complex).
4. Panic (simple and nasty).

Reply:
I think it is ok to just ignore invalid glob since user might provide valid glob which does not match anything. This simplifies implementation and user experience in my opinion would be the same.

Reply:
I've added logging on invocation (option 2). Lets discuss with @jessesuen when he arrives.

Reply:
Discussed with Jesse and logging is fine.